portal-impl/src/com/liferay/portlet/documentlibrary/service/permission/DLFileEntryPermission.java

```diff
@@ -110,6 +118,9 @@ public static boolean contains(
 			if (hasPermission != null) {
 				return hasPermission.booleanValue();
 			}
+			else if (actionId.equals(ActionKeys.VIEW)) {
```

Comment:
@adolfopa Yes, the code you're referring to ([this line specifically](https://github.com/liferay/liferay-portal/blob/master/portal-impl/src/com/liferay/portal/workflow/permission/WorkflowPermissionImpl.java#L90) is why we've done this; "VIEW" permissions in particular return null instead of false if Workflow is defined, so this is somewhat working around that. Do you think it should be done differently?